app/models/middleware_messaging.rb

```diff
@@ -3,4 +3,22 @@ class MiddlewareMessaging < ApplicationRecord
   belongs_to :middleware_server, :foreign_key => "server_id"
   acts_as_miq_taggable
   serialize :properties
+
+  include LiveMetricsMixin
+
+  def metrics_capture
+    @metrics_capture ||= ManageIQ::Providers::Hawkular::MiddlewareManager::LiveMetricsCapture.new(self)
+  end
+
+  def live_metrics_name
```

Comment:
I agree on moving `live_metrics_name` to `LiveMetricsMixin` and simplify the checks on that, but I'm not sure if we can do the same with other methods because the checks are on the `Performance` Mixin and I think is not safe to assume that all controllers that use performance will use `LIveMetricsMixin` too, or we can assume that at least for middleware?